Assets/Scripts/Scenes/InGame/Enemy/Trace/Feature/RadiationFeature.cs

```diff
@@ -1,16 +1,28 @@
+using System;
+using Cysharp.Threading.Tasks;
 using System.Threading;
 
+
 namespace Scenes.Ingame.Enemy.Trace.Feature
 {
     public class RadiationFeature : FeatureBase
     {
         CancellationTokenSource _cancellationTokenSource;
         FeatureView _view;
+        private int _change;
+        
 
         public override void Init(FeatureView view)
         {
             _cancellationTokenSource = new CancellationTokenSource();
             _view = view;
+            _view.TemperatureTrace.Subscribe(_ =>
+            {
+                _change = _view.stagetile.Msv + 10;
+                if (_change < 200)
+                    _change = 200;
+                _view.Msv(_change);
+            });
```

Comment:
```suggestion
            _view.OnTemperatureTrace
                .Skip(1)
                .Subscribe(_ =>
                {
                    _change = _view.stagetile.Msv + 10;
                    if (_change < 200)
                    {
                        _change = 200;
                    }
                    _view.Msv(_change);
                }).AddTo(view.gameObject);
```
Skipを入れないと、一番最初に何も入っていない状態でイベントが発行されエラーが置きます。
またAddToに関しては、このクラス自体がUnityにないためviewが破棄されたタイミングで破棄するようにすると、実装できました